src/SDKs/_metadata/databox_resource-manager.txt

```diff
@@ -0,0 +1,5 @@
+﻿Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/AzureDataBox/azure-rest-api-specs/blob/master/specification/databox/resource-manager/readme.md --csharp --csharp-sdks-folder=G:\AzureDataBox\azure-sdk-for-net\tools\..\src\SDKs\ --version=2.0.4215 --reflect-api-versions
+
+AutoRest code generation for https://github.com/AzureDataBox/azure-rest-api-specs/blob/master/specification/databox/resource-manager/readme.md failed. Please try again
```

Comment:
Looks like the code generation failed here, please regenerate the code and update PR